app/javascript/src/LoginPage/utils/formValidation.js

```diff
@@ -18,7 +18,9 @@ const constraintsTypes = {
   }
 }
 
-const validateForm = (form: SyntheticEvent<HTMLFormElement>, constraints: {}) => {
+type HTMLForm = SyntheticEvent<HTMLFormElement> | HTMLFormElement | HTMLElement | null
```

Comment:
Maybe just a naming problem but it looks weird that `validateForm` validates both forms and events. Is `validate.js` able to distinguish them?

Reply:
`validateForm` accepts a form element or an object, fixing